deploy/addons/addon-manager.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
This file is already present in deploy/static-pods/addon-manager, please copy that one to avoid duplication